includes/admin/post-types/post-tables/class-llms-admin-post-table-achievements.php

```diff
@@ -20,7 +20,7 @@
  */
 class LLMS_Admin_Post_Table_Achievements {
 
-	use LLMS_Trait_Award_Templates_Post_List_Table;
+	use LLMS_Trait_Award_Templates_Post_List_Table, LLMS_Trait_User_Engagement_Type;
```

Comment:
```suggestion
	use LLMS_Trait_Award_Templates_Post_List_Table;
	use LLMS_Trait_User_Engagement_Type;
```

I think I would prefer this notation for two _minor_ reasons:
1) that list might become long and, for readability, at a certain point will be forced to break on a new line, so let's do it by default
2) say I want to search for for all those classes which use ` LLMS_Trait_User_Engagement_Type`, the current notation makes it a bit harder, I should use reg exp at least.
What do you think?